frontend/src/components/StyledComponents.js

```diff
@@ -0,0 +1,59 @@
+import styled from 'styled-components/macro'
+export const MainContainer = styled.div`
+display: flex;
+flex-direction: column;
+@media (min-width: 600px) {
+  flex-direction: row;
+}
+`
+export const Container = styled.div`
+box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2), 0px 1px 1px 0px rgba(0, 0, 0, 0.14), 0px 1px 3px 0px rgba(0,0,0,.12);
+border-radius: 3px;
+width: 50%;
+background-image: linear-gradient(to right, #4CA3B0, #E1FAFA);
+padding: 20px;
+margin: 20px;
+@media (min-width: 600px) {
+  width: 25%;
+}
+`
```

Comment:
Would be nice with some spacing here! And styled components should be indented one level, like this:
```
import styled from 'styled-components/macro'

export const MainContainer = styled.div`
  display: flex;
  flex-direction: column;

  @media (min-width: 600px) {
    flex-direction: row;
  }
`

export const Container = styled.div`
  box-shadow: 0px 2px 1px -1px rgba(0, 0, 0, 0.2), 0px 1px 1px 0px rgba(0, 0, 0, 0.14), 0px 1px 3px 0px 
  rgba(0,0,0,.12);
  border-radius: 3px;
  width: 50%;
  background-image: linear-gradient(to right, #4CA3B0, #E1FAFA);
  padding: 20px;
  margin: 20px;

  @media (min-width: 600px) {
    width: 25%;
  }
`
```